scripts/v2/wait-for-operator-ready.sh

```diff
@@ -58,7 +58,7 @@ kubectl wait --for=condition=ready --timeout=3m pod -n azureserviceoperator-syst
 echo "Waiting for CRD cabundle..."
 export -f all_crds_have_cabundle
 export -f wait_for_crds_cabundle
-timeout 2m bash -c wait_for_crds_cabundle
+timeout 4m bash -c wait_for_crds_cabundle
```

Comment:
This seems excessive to me. We shouldn't still be seeing this take that long on the beefy pool... is it?